train-tachograph/src/main/java/hu/bme/mit/train/tachograph/TrainTachograph.java

```diff
@@ -0,0 +1,27 @@
+package hu.bme.mit.train.tachograph;
+
+import com.google.common.collect.HashBasedTable;
+import com.google.common.collect.Table;
+import hu.bme.mit.train.controller.TrainControllerImpl;
+import hu.bme.mit.train.interfaces.TrainController;
+import hu.bme.mit.train.interfaces.TrainUser;
+import hu.bme.mit.train.user.TrainUserImpl;
+
+import java.time.LocalTime;
+
+
+public class TrainTachograph {
+
+    private TrainController controller = new TrainControllerImpl();
+    private TrainUser user = new TrainUserImpl(controller);
+    private Table<LocalTime, Integer, Integer> table;
+
+    public TrainTachograph() {
+        table = HashBasedTable.create();
+        table.put(java.time.LocalTime.now(), user.getJoystickPosition(), controller.getReferenceSpeed());
+    }
+
+    public boolean TableIsEmpty(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.